src/Raven.Server/Documents/Sharding/Subscriptions/OrchestratedSubscriptionConnection.cs

```diff
@@ -35,6 +38,7 @@ public class OrchestratedSubscriptionConnection : SubscriptionConnectionBase<Orc
         {
             _databaseContext = tcpConnection.DatabaseContext;
             _tokenRegisterDisposable = CancellationTokenSource.Token.Register(() => _processor?.CurrentBatch?.SetCancel());
+            _dbIdToRemove = new HashSet<string>() { _databaseContext.DatabaseRecord.Sharding.DatabaseId };
```

Comment:
sanity-check: no need for case insensitive comparer here?

Reply:
added